README.md

```diff
@@ -59,10 +59,11 @@
 - (Optional) Install a third-party antivirus product of your choice.
 - Activate Windows using a valid license key. If you don't already have one, ask HR about getting one.
 - Install all available Microsoft updates, restarting your computer as necessary.
+- Open PowerShell with Administrator privileges. Run `Set-ExecutionPolicy RemoteSigned`. Type Y and press Enter to confirm that you want to make this change.
 - Install the [Chocolatey package manager for Windows](https://chocolatey.org/install).
-- Open either PowerShell or Command Prompt (your choice) with Administrator privileges. Run, or copy and paste, the [install-apps-via-chocolatey.cmd](./install-apps-via-chocolatey.cmd) script to install a number of programs you will probably need. (Feel free to peek at this file's contents to see what it contains.)
-- From the same PowerShell or Command Prompt window, run the command `shutup10`. Here, you can easily configure a number of settings to have Windows respect your privacy a little more.
-- After running shutup10, restart Windows if necessary.
+- Reopen PowerShell with Administrator privileges, if necessary. Run the [install-apps-via-chocolatey.ps1](./install-apps-via-chocolatey.ps1) script to install a number of programs you will probably need. (Feel free to peek at this file's contents to see what it contains.)
+- From the same PowerShell window, run the command `oosu10`. Here, you can easily configure a number of settings to have Windows respect your privacy a little more.
```

Comment:
Sorry @stephengtuggy, while **I do** respect your values towards privacy, I have reservations about the utility of this.  As these are Shift3 work PCs, we need stability and productivity, foremost.

Here is `oosu`...
https://www.oo-software.com/en/shutup10

At a cursory glance, this really seems like it'll hamper your Windows experience.

https://www.howtogeek.com/273513/why-you-shouldnt-use-anti-spying-tools-for-windows-10/

Everyone here, IMO, is capable of exploring the settings made available through Window's _extensive_ setup screens, without running the risk of breakage and OMG WINDOWS! events in the future which are actually caused by system invasive mystery scripts.

Reply:
@dunlavy I haven't experienced **any** of the problems listed in that howtogeek article. The only valid criticism of shutup10 I found in the article was that the tool isn't as effective as it might claim. But I already addressed that in the README changes in question in this PR. (I said "to have Windows respect your privacy **a little** more.")

Sure, you **can** use shutup10 to disable Windows Update and a number of other important Windows features. (You can use `regedit` to do the same thing.) But even shutup10 itself recommends against doing the above. Really, it just provides a UI to let you configure privacy-related settings in one place -- many of which do not come with a built-in Windows UI for changing them.

Everyone here, IMHO, is capable of reading the documentation for each setting in shutup10, and determining for themselves which ones (if any) they want to change.

I can make the "running oosu10" step optional, if you would like. It's not really required, just a good idea to run, in my opinion.

Reply:
Just consider by making this S&P, you are owning it.  Any thoughts / suggestions from other reviewers are welcome!

Maybe make this an optional bit, so you can insert some of your preferences but without making it a fully owned Shift3-condoned practice.

Reply:
> Any thoughts / suggestions from other reviewers are welcome!

We can't exactly solicit thoughts if the conversations are resolved.

Reply:
@dunlavy those are good points. 

I see that @stephengtuggy updated the message saying that this is optional and that you should read about it before you use it. I say that resolves this issue, doesn't it?